src/main/resources/templates/city_show.html

```diff
@@ -13,5 +13,18 @@ <h2>City</h2>
 Population: <span th:inline="text"> [[${city.population}]]</span><br/>
 Local Weather: <span th:inline="text"> [[${city.temp}]]</span> Degrees F<br/>
 LocalTime: <span th:inline="text"> [[${city.time}]]</span><br/>
+<br />
+For a fabulous weekend get away for 2 to [[${city.name}]] sign up here:
+<form action="#" th:action="@{/cities/reservation}" method="post">
+  Email: <input type="text" th:name="email"/><br />
+  <input type="radio" th:value="luxury" th:name="level">
```

Comment:
Pretty good and straight forward. I don't know how strict the teacher will be with as he used "moderate" & "budget" instead for the values. But your code is pretty good that this is the only thing I can comment on